src/Blocks/BlockIce.h

```diff
@@ -25,9 +25,54 @@ class cBlockIceHandler :
 		{
 			return cItem(m_BlockType);
 		}
```

Comment:
Now I am confused

Reply:
diff looks weird?

Reply:
I ment to mark then line with ````{{``` why those are needed

Reply:
The std::array {{?

I think the first { is to indicate initialisation of std::array, the second { indicates you want to aggregate-initialise the C-array the std::array contains, and the remaining {x,y,z} initialises the contained Vector3i

Reply:
ok - thank you